frontend/public/components/overview/build-configs-overview.tsx

```diff
@@ -0,0 +1,88 @@
+import * as _ from 'lodash-es';
+import * as React from 'react';
+import { Link } from 'react-router-dom';
+import { Button, ListGroup } from 'patternfly-react';
+
+import { BuildPhaseIcon } from '../build';
+import { errorModal } from '../modals/error-modal';
+import { startBuild } from '../../module/k8s/builds';
+import { fromNow } from '../utils/datetime';
+import { ResourceLink, resourcePath, SidebarSectionHeading } from '../utils';
+
+const conjugateBuildPhase = (phase) => {
+  switch (phase) {
+    case 'Cancelled':
+      return 'was cancelled';
+    case 'Error':
+      return 'encountered an error';
+    case 'Failed':
+      return 'failed';
+    default:
+      return `is ${_.toLower(phase)}`;
+  }
+};
+
+const BuildOverviewItem = ({build}) => {
+  const {metadata} = build;
+  const timestamp = build.status.completionTimestamp || build.status.startTimestamp;
```

Comment:
I think these could both be unset for new builds that haven't started running. (Not uncommon for builds with serial policy.)

Reply:
Should we add the creation time also? That way new builds will also have a relative time shown.

Reply:
> Should we add the creation time also? That way new builds will also have a relative time shown.

Yeah, I think so.

Reply:
Maybe change the variable name to `lastUpdated`?